examples/controlnet/train.sh

```diff
@@ -0,0 +1,35 @@
+###controlnet.sh PSEUDO###
+env_name=${1}
+model_name=${2}
+batch_size=${3}
+output_dir=${4:-"outputs"}
+current_dir=$(pwd)
+log_dir=${5:-"${current_dir}/logs/${model_name}.json"}
+
+# Check if $batch_size is provided , if not use default value of 5
+if expr "$batch_size" + 0 > /dev/null 2>&1; then
+  batch_size=$batch_size
+else
+  batch_size=5
+fi
+
+# Activate conda env
+base_env=$(conda info | grep -i 'base environment' | awk -F': ' '{print $2}' | sed 's/ (read only)//' | tr -d ' ')
+source ${base_env}/etc/profile.d/conda.sh
+conda activate ${env_name}
+moreh-switch-model -M 1 
+
+# Run training script
+echo "# ========================================================= #"
+echo "training ${model_name}.."
+python train_controlnet.py \
```

Comment:
Activating `conda` environment in a shell script is (sort of) a hassle. I suggest running `train_controlnet.py` by using `conda run`. For example: `conda run -n ${env_name} python3 train_controlnet.py ...`